### DISCUSSION TITLE
Move a boat with speed and orientation

### DISCUSSION THREAD

**Discussion by yannickModulo** [jrouwe/JoltPhysics#953] [Q&A] [answered]
Hi, 

We are starting to use JoltPhysics to control a boat in the sea. Do I need to use a Vehicle class to apply the speed and rotation angle of the boat ?  I am a little bit lost to start.

**Comment by jrouwe** [comment]
I added a demo here: https://github.com/jrouwe/JoltPhysics/commit/7da4f39fa65c3b592cf3b2496f92417f2d1fe31e

**Reply by yannickModulo** [reply, marked as answer]
Thanks a lot!
